packages/editor/src/editor.spec.ts

```diff
@@ -199,6 +221,54 @@ describe('Editor', () => {
     expect(content).to.be.equal('1 + 1');
   });
 
-  // TODO: e2e or mock tests for _getExtension
-  // TODO: Write a node script that acts as an external editor and write e2e tests
+  context('runEditCommand', () => {
+    let externalEditor: string;
+
+    beforeEach(async() => {
+      externalEditor = await setupExternalEditor(base);
+    });
+
+    afterEach(async() => {
+      await fs.unlink(externalEditor);
+    });
+
+    it('returns an error if editor is not defined', async() => {
+      cmd = null;
+      delete process.env.EDITOR;
```

Comment:
Anything with side effects should probably go into before/after hooks, in general